UnityGLTF/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.5f1
+m_EditorVersion: 5.6.1f1
```

Comment:
You should probably revert this. It's going back in version?

Reply:
This was back in the first PR I made, which wasn't organized at all. My bad. There is probably more stuff to revert (only a couple of scripts should be edited...)